bb8

```diff
@@ -13,6 +13,7 @@ docker run \
     --rm \
     -v /var/run/docker.sock:/var/run/docker.sock \
     -v bb8_ssh:/bb8/etc/.ssh \
+    -v /etc/machine-id:/bb8/etc/machine-id \
```

Comment:
Looking at this and @MartinEden's comment below, another option would be to build this in by changing `setup` to say:

```
machine_id=$(cat /etc/machine-id)
docker build --build-arg "TARGETS=$targets" --build-arg "MACHINE_ID=$machine_id" --tag bb8 .
```

Then in the dockerfile go with

```
ARG MACHINE_ID
RUN echo $MACHINE_ID > /etc/machine-id
```

after which the conditionals can be removed from the settings? Not sure if it's better...

Reply:
Actually the conditional still couldn't be removed from the settings, the condition would just remain as is!

Reply:
Ah, probably best to leave it as you have it then - that's how the ssh bits get in already...

Reply:
I do like this more. Is there a way to easily inspect build args in a built image (that's not currently running as a container)? Otherwise this gets a bit hard to inspect compared to just `cat /etc/machine-id`).

Reply:
Probably what this really indicates is that we shouldn't be using the same settings file for setup and running, given that we expect them to have a different structure. And indeed should probably re-think the whole way we configure it to use a volume for target config the way we do for logs and ssh config.

Reply:
just looking at the output of `docker image inspect` at the moment - I think labels are what we should use if we want to inspect images at rest and I think we can use [`LABEL`](https://docs.docker.com/engine/reference/builder/#label) to do this.  Otherwise no sign of build args I used on the image I'm looking at

Reply:
In that case, I agree, let's leave it as you have written it for now.